WordPress/gutenberg
# Shadow Panel: Improve a11y and fix browser console error

## What?

This PR makes the following two changes in the shadow editing panel.

1. Resolve browser warning errors regarding `ariaExpanded`
  ![image](https://github.com/WordPress/gutenberg/assets/54422211/65c338f1-df73-4c48-83bc-1f9180557fe6)
2. Remove invalid `arialabel` attributes
  ![arialabel](https://github.com/WordPress/gutenberg/assets/54422211/342954ee-ba64-4c6f-ab61-0f53f5e7e60e)

## Why? / How?

> Resolve browser warning errors regarding `ariaExpanded`

`aria-*` HTML should be hyphen-cased ([Source](https://legacy.reactjs.org/docs/accessibility.html#wai-aria)).

> Remove invalid `arialabel` attributes

This attribute itself should not be needed, since the button has proper text.

## Testing Instructions

- Go to the Site Editor > Global Styles > Shadows > Natural.
- Confirm that no browser warnings are displayed.
- Confirm that the toggle button's `aria-expanded` attribute toggles when the shadow popup is opened and closed.
